app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -284,6 +284,9 @@ public boolean onLongClick(View v) {
                     mediaAdapter.notifyItemChanged(toggleSelectPhoto(m));
                     editMode = true;
                 }
+                else {
```

Comment:
@angmas1 make this if else block similar to the block in lines 277-280. There is no need for the braces as there is a single statement in the else block.